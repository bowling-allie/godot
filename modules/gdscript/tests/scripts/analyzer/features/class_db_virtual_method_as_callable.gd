class TestClass:
	pass

func test() -> void:
	var free_callable: Callable = free
	print(free_callable)
	var test_class_new_callable: Callable = TestClass.new
	print(test_class_new_callable)
