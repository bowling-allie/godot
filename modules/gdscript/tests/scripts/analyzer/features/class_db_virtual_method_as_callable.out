GDTEST_OK
RefCounted::free
GDScript::new
